Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -160,6 +176,11 @@ - (void)timeout:(NSTimer *)timer
 
 #pragma mark - Public API
 
+RCT_EXPORT_METHOD(setRNConfiguration:(RCTLocationConfiguration)config)
```

Comment:
Why the RN in the name? Can this be just `setConfiguration:`

Reply:
Sorry, missed that in your original description. I think it's a good idea to avoid potential conflicts with the web spec. We can name the native module however we want though, so let's call the native method `setConfiguration` and the API on geolocation `setReactConfiguration`?